bonsai/Assets.xcassets/Category/cross.vial.fill.imageset/Contents.json

```diff
@@ -0,0 +1,22 @@
+{
+  "images" : [
+    {
+      "filename" : "cross.vial.fill.pdf",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "cross.vial.fill-1.pdf",
+      "idiom" : "universal",
+      "scale" : "2x"
```

Comment:
for pdf we can simply specify universal or single scale

Reply:
ok